### DISCUSSION TITLE
Unable to perform any changes in the UI

### DISCUSSION THREAD

**Discussion by Artur-Galstyan** [pocketbase/pocketbase#1441] [Q&A] [answered]
Hi everyone,

this is my first issue, so I'm sorry if I forget about some guidelines.

The issue is that I'm unable to perform any changes in the UI. For example, I've created a table and then wanted to update the API rules but whatever I change (just letting the rules be empty as prompted by the UI) I'm getting this error message:

Something went wrong while processing your request.

And then checking the console to see if there's anything it says:

PATCH http://[my linode IP]/api/collections/mnfnwd2c904v47n 400 (Bad request: request method denied)

I was following Fireship's tutorial and created a linode server on which Pocketbase is running. 

The version I'm using is [v0.10.4 Release](https://github.com/pocketbase/pocketbase/releases/tag/v0.10.4)

I also tried changing the applition URL under settings -> application -> application URL but I'm getting the same error :(

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not able to reproduce the issue. There are several things you could try:

- Open your browser's "Inspect tool > Network", find the 400 request and check the request/response tabs for a more detailed error message.

- Check if there is an error log in the "Admin UI > Logs" when trying to save the collection.

- Start PocketBase with `--debug` (eg. `./pocketbase serve --debug`) and check the server logs.


> I also tried changing the applition URL under settings -> application -> application URL but I'm getting the same error :(

The Application url and name are currently used primarily in the mail templates. It shouldn't have effect on the web API.

**Reply by Artur-Galstyan** [reply]
Thanks, wow. WatchGuard blocked the requests:

Request denied by WatchGuard HTTP Proxy.
Reason: request method denied method='PATCH'
Please contact your administrator for assistance.
Path: /api/collections/mnfnwd2c904v47n
